lua/telescope/make_entry.lua

```diff
@@ -294,15 +294,25 @@ function make_entry.gen_from_symbols(opts)
   opts = opts or {}
   opts.tail_path = get_default(opts.tail_path, true)
 
-    local displayer = entry_display.create {
+  local displayer = entry_display.create {
     separator = "",
+    hl_chars = { ['['] = 'TelescopeBorder', [']'] = 'TelescopeBorder' },
     items = {
       { width = 6 },
       { width = 40 },
-      { width = 1 },
       { remaining = true },
-      { width = 1 },
-    },
+    }
+  }
+
+  local default_type_highlight = {
```

Comment:
Btw i moved that table out. We might wanna have that outside of this scope as well so it won't get redefined that often.